app/src/main/java/com/onboarding/parkingsystemjava/mvp/view/SetParkingView.java

```diff
@@ -0,0 +1,34 @@
+package com.onboarding.parkingsystemjava.mvp.view;
+
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.onboarding.parkingsystemjava.R;
+import com.onboarding.parkingsystemjava.fragment.SetParkingLotsDialog;
+import com.onboarding.parkingsystemjava.mvp.contract.SetParkingContract;
+
+import java.lang.ref.WeakReference;
+
+public class SetParkingView implements SetParkingContract.SetParkingView {
+    private WeakReference<SetParkingLotsDialog> fragmentRef;
+
+    public SetParkingView(SetParkingLotsDialog fragment) {
+        this.fragmentRef = new WeakReference<>(fragment);
+    }
+
+    @Override
+    public String getLots() {
+        TextView t = fragmentRef.get().getView().findViewById(R.id.edit_text_dialog_fragment_parking_input);
```

Comment:
please avoid using one letter var names

Reply:
also you should use viewBinding

Reply:
switched to viewBinding, removed variable t